gobblin-docker/gobblin-wikipedia/ubuntu-gobblin-0.7.0/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gobblin/gobblin-distributions:ubuntu-gobblin-0.7.0
+
+ENV GOBBLIN_WORK_DIR /home/gobblin/work-dir
+
+RUN echo "Downloading Gobblin Wikipedia Configuration Files"
+RUN mkdir -p /etc/opt/gobblin/job-conf
+
+# Downloading wikipedia.pull
+RUN curl -L --progress-bar https://raw.githubusercontent.com/linkedin/gobblin/master/gobblin-example/src/main/resources/wikipedia.pull -o /etc/opt/gobblin/job-conf/wikipedia.pull
```

Comment:
wikipedia.pull might change between different Gobblin versions. Isn't it bundled in the gobblin-distribution tarball? If so, we can just copy it from there to be sure that it would work.